### ISSUE TITLE
`hashivault_read_to_file` - compatibel with kv v2?

### ISSUE THREAD

**Issue by pat-s** [TerryHowe/ansible-modules-hashivault#392]
`hashivault_read` is [compatibel with kv v2](https://terryhowe.github.io/ansible-modules-hashivault/modules/hashivault_read_module.html#hashivault-read-module) and I can read successfully read secrets.

I am getting 

```
"Error Forbidden(1 error occurred:\n\t* permission denied\n\n, 
```

on the same paths when trying to use `hashivault_read_to_file` though. Does is support kv v2? I could not find any indication in favor or against it in [its documentation](https://terryhowe.github.io/ansible-modules-hashivault/modules/hashivault_read_to_file_module.html#hashivault-read-to-file-module).

**Comment by TerryHowe** [comment]
Are you looking for the `version` option https://terryhowe.github.io/ansible-modules-hashivault/modules/hashivault_read_module.html#hashivault-read-module ?

**Comment by pat-s** [comment]
Yes 👍 it's missing, hence it's not supported for this module yet and only works with v1?
